src/commands/util/ping.js

```diff
@@ -1,10 +1,33 @@
-const { SlashCommandBuilder } = require("discord.js");
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
 
 module.exports = {
+
+	//Creating the command and description.
 	data: new SlashCommandBuilder()
 		.setName("ping")
 		.setDescription("Replies with Pong!"),
+	
+	//Functionality of the command.
 	async execute(interaction) {
-		await interaction.reply("Pong!");
+		await interaction.reply({ 
+			content: 'Calculating Ping...' 
+		});
+
+		const reply = await interaction.fetchReply();
+		const botLatency = reply.createdTimestamp - interaction.createdTimestamp;
+		const apiLatency = Math.round(interaction.client.ws.ping);
+		
+		const pingEmbed = new EmbedBuilder()
+			.setColor('#4fffa4')
+			.setTitle('🏓 Pong!')
+			.addFields(
+				{name: 'Bot Latency', value: `**${botLatency}ms**`},
+				{name: 'API Latency', value: `**${apiLatency}ms**`},
+			)
+
+		return await interaction.editReply({
```

Comment:
remove return & re-test w/o the return it should function the same

Reply:
Did not work... have to keep the return.

Reply:
Hey uh turns out you were correct hee hee xD